src/applications/calibrate/app_vCalib.xml.template

```diff
@@ -12,7 +12,7 @@
 
 <module>
     <name> vPreProcess </name>
-    <parameters> --flipx --flipy --pepper false --split </parameters>
+    <parameters> --flipx --flipy --pepper false --split_stereo </parameters>
```

Comment:
⚠️ These updates aren't really to do with the Pull Request so shouldn't be in this PR, (but could have been in the last one about the xml updates)

Reply:
Keep them here for now